### PR TITLE
Added NEFFEX/Fight Back: The Collection/Fight Back

### PR DESCRIPTION
**Pull request by TheBinaryLoop**

### CODE REVIEW

File: database/N/NEFFEX/Fight Back: The Collection/Fight Back [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+__________________________________
+Language          American English
+Name              Fight Back
+Artist            NEFFEX
+Album             Fight Back: The Collection
+Track no          1
+Year              2018
+MusicBrainz ID    cce1780b-3fc0-42bc-8093-f9a99c6e331a
+Original text by  Brandon Horth/Cameron Wales
```
Comment (snshn) [inline database/N/NEFFEX/Fight Back: The Collection/Fight Back:72]: Could you please write authors of the original text as: `Brandon Horth / Cameron Wales` or `Brandon Horth,  Cameron Wales` (whichever you think fits better)?

File: database/N/NEFFEX/Fight Back: The Collection/Fight Back [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+Make them take it all back
+Don't tell me you believe that
+Are you just gonna take that?
+
+Or will you fucking fight back?
+Or will you fucking fight back?
+
+
+__________________________________
+Language          American English
```
Comment (snshn) [inline database/N/NEFFEX/Fight Back: The Collection/Fight Back:65]: I usually put Language below Year, easier to read metadata for humans that way.

File: database/N/NEFFEX/Fight Back: The Collection/Fight Back [hunk 3]
```diff
@@ -0,0 +1,72 @@
+You're never gonna make it, you're not good enough
```
Comment (snshn) [inline database/N/NEFFEX/Fight Back: The Collection/Fight Back:1]: Could you please change the `'` signs to proper apostrophe signs?  More info here: https://smartquotesforsmartpeople.com

**Review by snshn — CHANGES_REQUESTED** [review]
The name of the album contains a colon, the cloning of this repo may fail on Windows... let's run the workflow and see if it works or not

**Comment by snshn** [comment]
Yeah, Windows CI fails because of that colon... the conventional way to deal about that type of names is usually to rename to something more filesystem-friendly, e.g. `Fight Back - The Collection`, but the name in the metadata of course can stay the same, and that's how it's going to be displayed in HTML on the website, with colon.

**Comment by TheBinaryLoop** [comment]
Changes were made as requested

**Review by snshn — APPROVED** [review]

**Comment by snshn** [comment]
Hey, thank you for this contribution Lukas!

**Pull request merged**